alchemy/test/cloudflare/worker.test.ts

```diff
@@ -1356,4 +1356,240 @@ describe("Worker Resource", () => {
       await destroy(scope);
     }
   });
+
+  test("create worker with Workers for Platform", async (scope) => {
+    const workerName = `${BRANCH_PREFIX}-test-worker-platform`;
+
+    let worker: Worker | undefined;
+    try {
+      // Create a worker for Workers for Platform
+      worker = await Worker(workerName, {
+        name: workerName,
+        script: `
+          export default {
+            async fetch(request, env, ctx) {
+              return new Response('Hello from Workers for Platform!', {
+                status: 200,
+                headers: { 'Content-Type': 'text/plain' }
+              });
+            }
+          };
+        `,
+        format: "esm",
+        platform: true,
+      });
+
+      expect(worker.id).toBeTruthy();
+      expect(worker.name).toEqual(workerName);
+      expect(worker.platform).toEqual(true);
+      expect(worker.format).toEqual("esm");
+
+      // Update the worker to verify platform option is preserved
+      worker = await Worker(workerName, {
+        name: workerName,
+        script: `
+          export default {
+            async fetch(request, env, ctx) {
+              return new Response('Updated Workers for Platform!', {
+                status: 200,
+                headers: { 'Content-Type': 'text/plain' }
+              });
+            }
+          };
+        `,
+        format: "esm",
+        platform: true,
+      });
+
+      expect(worker.platform).toEqual(true);
+    } finally {
+      await destroy(scope);
+      await assertWorkerDoesNotExist(workerName);
+    }
+  });
+
+  test("Workers for Platform supports Durable Object migrations", async (scope) => {
```

Comment:
I wonder if we can reuse the tests  we have right now except add a helper that calls the same test handler once for non-wfp and once for wfp

It wouldn't be applicable to all tests and should be opt in, but I think that might be more scalable that duplicating the tests by hand. 

testBothPlatforms([true, false], "it should ..", async (scope, isWFP) => ..

It would automatically add a suffix -wfp when true and call the handler. 

All our tests that deploy workers and especially the ones that test bindings could import and use this to quickly test the entire surface area with WFP.